packages/squiggle-lang/src/js/math.ts

```diff
@@ -0,0 +1,117 @@
+type SampleArgs = {
+  probs: number[];
+  size: number;
+};
```

Comment:
I don't expect this would be used anywhere outside of `random_sample`, so I'd just inline it.
(In Rescript it wasn't inlined because Rescript doesn't allow inline record types).